### ISSUE TITLE
error: unknown long flag '--web.config.file'

### ISSUE THREAD

**Issue by chillriver** [Lusitaniae/apache_exporter#130]
Hello,

how does the topic already say. I have that error when I started the service from systemd an read in the log from journalctl:

`apache_exporter: error: unknown long flag '--web.config.file', try --help`

I want to add a configfile  in order to use TLS and define the certspath. Here is my configfragment in the unitfile from systemd.

ExecStart=/usr/local/bin/apache_exporter \
--web.config.file="/etc/apache_exporter/web.yml" \
--scrape_uri="http://localhost/server-status?auto" \
--telemetry.address="0.0.0.0:9117"

Version of my apache_exporter v0.11.0

**Comment by Lusitaniae** [comment]
hi @chillriver, that's currently not a supported option

Feel free to submit a PR in order to add that option

**Comment by Lusitaniae** [comment]
hi @chillriver, that's currently not a supported option

Feel free to submit a PR in order to add that feature

**Comment by Lusitaniae** [comment]
That's good to hear

Seems I rushed in my reply and give you wrong information!

Would you like to make a quick PR to update the parameter mentioned in the README at https://github.com/Lusitaniae/apache_exporter/blob/aee05657dae4661965fbd40df3fb99fc43b5c016/README.md#tls-and-basic-authentication and anything else you think relevant?

**Comment by chillriver** [comment]
Yes I can edit the README. It should be change the TLS chapter and also the help on flags.

**Comment by Lusitaniae** [comment]
Much appreciated!
